.github/workflows/build.yml

```diff
@@ -84,7 +84,7 @@ jobs:
     - name: Install Deps
       run: |
         dnf install -y --setopt=install_weak_deps=False \
-          git gcc-c++ cmake rpm-build xml-security-c-devel zlib-devel vim-common doxygen boost-test swig python3-devel java-1.8.0-openjdk-devel xsd
+          git gcc-c++ cmake rpm-build xml-security-c-devel zlib-devel vim-common doxygen boost-test swig python3-devel java-1.8.0-openjdk-devel xsd minizip-devel
```

Comment:
Isn't that minizip-ng-devel?
https://packages.fedoraproject.org/pkgs/minizip-ng/minizip-ng-devel/

Reply:
No because once we switch to Fedora 40, the minizip-ng-devel is a different package than the minizip-ng-compat-devel.
When the minizip-devel is used it will be provided by `minizip-ng-devel` for Fedoras until 40 (which is okay, because it was wrongly named until that point, it should have been named minizip-ng-compat-devel) and since 40 it will be provided by `minizip-ng-compat-devel`.